helix-view/src/editor.rs

```diff
@@ -571,7 +577,30 @@ impl Editor {
         self.status_msg = Some((error.into(), Severity::Error));
     }
 
+    pub fn unset_theme_preview(&mut self) {
+        if let Some(last_theme) = self.last_theme.take() {
+            let theme_name = &(*last_theme).name;
+            match self.theme_loader.load(theme_name) {
+                Ok(theme) => {
+                    self.set_theme(theme);
+                }
```

Comment:
No need to load since now we have the whole theme already, can just set theme.

Reply:
ahhh, silly mistake, will fix